extensions/VisualEditor/wikia/VisualEditor.i18n.php

```diff
@@ -38,6 +38,8 @@
 	'wikia-visualeditor-dialogbutton-transclusion-tooltip' => 'Template',
 	'wikia-visualeditor-dialogbutton-infobox-tooltip' => 'Infobox',
 	'wikia-visualeditor-dialog-infobox-insert-title' => 'Select infobox to insert',
+	'wikia-visualeditor-dialog-infobox-insert-empty-state' => 'Sorry. No infoboxes found',
```

Comment:
just a thought - maybe should we mention here we are talking about the portable infoboxes only (and we don't care about the old ones) ?
AFAIR one of the users mentioned (in thread) that it was a surprise for him, that the new infobox edit tool in VE doesn't support the old ones.